src/liboslexec/closure.cpp

```diff
@@ -76,7 +76,7 @@ print_component_value(std::ostream &out, ShadingSystemImpl *ss,
     else if (type == TypeDesc::TypeVector)
         out << "(" << ((Vec3 *)data)->x << ", " << ((Vec3 *)data)->y << ", " << ((Vec3 *)data)->z << ")";
     else if (type == TypeDesc::TypeString)
-        out << "\"" << ((ustring *)data)->c_str() << "\"";
+        out << "\"" << ((ustring *)data)->string() << "\"";
```

Comment:
Good call, I didn't realize that overload was there. I'll amend this on my next pass through the code.